docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -141,7 +177,7 @@ kubectl logs pod/hazelcast-sample-0
 oc logs pod/hazelcast-sample-0
 ----
 
-You should see the following:
+If the cluster is running, the result is similar to the following:
```

Comment:
It is not remotely similar to the following!

Reply:
I didn't quite understand your point, could you please provide more details?